test/add_remove_bench.jl

```diff
@@ -0,0 +1,34 @@
+using StingerWrapper
+using Base.Test
+
+function generate!(edges, numvertices::Integer, numedges::Integer)
+    for i in 1:numedges
+        src, dst = rand(0:numvertices-1), rand(0:numvertices-1)
+        edges[:,i] = [0, src, dst, 1, 0]
+    end
+end
+
+function bench(numvertices, numedges)
+    info("Allocating a STINGER")
+    s = Stinger()
+    edges = zeros(Int64, 5, numedges)
+    info("Generating the edges")
+    @time generate!(edges, numvertices, numedges)
+    info("Inserting $(size(edges,2)) edges")
+    @time insert_edges!(s, edges, numedges)
+    @test consistency_check(s, numvertices) == true
+    removals = edges[:,randperm(numedges)]
+    removals = unique(removals, 2)
+    @show size(removals)
+    info("Removing 1/8 of the edges")
+    @time remove_edges!(s, removals, ceil(Int, numedges/8))
+    @test consistency_check(s, numvertices) == true
+    return s, edges, removals
+end
+
+info("Setting STINGER Parameters")
+ENV["STINGER_MAX_MEMSIZE"] = 10000000
```

Comment:
Should we change this to stick to the amount of memory needed for the default configuration of STINGER? It might be confusing for users to get the error and the abort.